frontend/src/styles/components/commentlist.css

```diff
@@ -0,0 +1,25 @@
+/* Adjustments for toggleable comments */
+.commentlist-heading {
+  font-family: 'Nunito', sans-serif;
+  font-size: var(--font-size-medium);
+  color: var(--color-red-brown);
+}
+
+.commentlist-heading::after {
+  display: inline-block;
+  align-self: center;
+  font-size: 12px;
+  color: rgba(0, 0, 0, 0.55);
+  font-family: 'Nunito', sans-serif;
+  font-size: var(--font-size-medium);
+  color: var(--color-red-brown);
+  margin-top: var(--space-small);
+  margin-bottom: var(--space-small);
+}
+
+details.CommentList[open] .commentlist-heading::after {
+  content: 'Hide comments';
+}
+details.CommentList:not([open]) .commentlist-heading::after {
+  content: 'Show comments';
```

Comment:
I suggest adding the number of comments when commentlist is not open. so it can say something like "comments 34" or something like that. So we can in feature work say this functionality could be expanded to show only latest comments or highly liked ones. For example of how to do this in js, react please see this screenshot taken from [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details)
![Screenshot 2021-05-15 at 08 06 19](https://user-images.githubusercontent.com/43238452/118349978-746b4700-b554-11eb-9166-9bf5ebbe6467.png)

Reply:
Thank you for your suggestion. I will put this as an issue now and address it later.